### DISCUSSION TITLE
dateAndTime property

### DISCUSSION THREAD

**Discussion by Inoy-D** [zwave-js/zwave-js-ui#2277] [General]
What input format dateAndTime configuration property?

**Comment by robertsLando** [comment]
Could you better explain what you mean?

**Comment by Inoy-D** [comment]
![Q](https://user-images.githubusercontent.com/73270846/154832373-a8bc30dc-cf28-474a-baa8-c0c850ca9cf7.jpg)

**Reply by robertsLando** [reply]
@AlCalzone Couldn't you try to parse provided value as a time string?

**Reply by AlCalzone** [reply]
Please open an issue in node-zwave-js so I can check how to make this field more intuitive.

**Reply by robertsLando** [reply]
https://github.com/zwave-js/node-zwave-js/issues/4265
